api/bin/chainflip-broker-api/src/main.rs

```diff
@@ -100,6 +106,17 @@ pub trait Rpc {
 		affiliate_fees: Option<Affiliates<AccountId32>>,
 		dca_parameters: Option<DcaParameters>,
 	) -> RpcResult<SwapPayload>;
+
+	#[method(name = "mark_btc_transaction_as_tainted", aliases = ["broker_markBtcTransactionAsTainted"])]
```

Comment:
Can we make this more general, just to future-proof it? 

We could use an enum to make it a little less painful:

```rust
enum TxId {
	Bitcoin {
		tx_id: cf_chains::btc::Hash,
	}
	// other variants reserved for other chains.
}

fn mark_transaction_as_tainted(&self, tx_id: TxId) {
	// ...
}
```

Reply:
(Same for the other methods)